docs/designs/sanitize-html.md

```diff
@@ -26,10 +27,11 @@ The sanitization is performed against user HTML, that is in most cases HTML tags
 
 ### Sanitization Modes
 
-There are 3 sanitization modes:
+There are 4 sanitization modes:
 
 - `standard` (*default*): Described below
 - `strict`: Disallow html
+- `strict-inline`: Based on strict, permitting only phrasing content. Block elements such as `blockquote`, `ul`, and `p` are not permitted.
```

Comment:
Today we have `contentType: inlineMarkdown`, if that implies disallowing all block elements, is `strick-inline` equivalent to:

```jsonc
{
    "contentType": "inlineMarkdown",
    "sanitizationMode": "standard", // the default sanitization mode, permit inline elements such as `<a>`, `<strong>`
}
```